crates/sdk/src/tracing.rs

```diff
@@ -13,9 +14,10 @@ pub fn init_tracing(
     service_name: Option<&str>,
     otlp_endpoint: Option<&str>,
 ) -> Result<(), Box<dyn Error + Send + Sync>> {
-    opentelemetry::global::set_text_map_propagator(
-        opentelemetry_sdk::propagation::TraceContextPropagator::new(),
-    );
+    opentelemetry::global::set_text_map_propagator(TextMapCompositePropagator::new(vec![
+        Box::new(opentelemetry_sdk::propagation::TraceContextPropagator::new()),
+        Box::new(opentelemetry_zipkin::Propagator::new()),
```

Comment:
Shouldn't the Zipkin one go first?

Reply:
No, Zipkin should go last. The composite propagator gets the latest propagation value.
https://github.com/open-telemetry/opentelemetry-rust/blob/2286378632d498ce4b2da109e5aa131b34ae1a8f/opentelemetry/src/propagation/composite.rs#L106

Reply:
Well, that is confusing. Thanks for clarifying.